### PR TITLE
Add support for Chinese and Japanese Pokémon names

### PR DESCRIPTION
**Pull request by cherry-geqi**

### CODE REVIEW

**Comment by sindresorhus** [comment]
Awesome! :)

According to https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes it should be:
`jp` => `ja`
`zh-CN` => `zh`.

File: test.js [hunk 1]
```diff
@@ -58,3 +70,15 @@ test('Get ID by German name', testIDs, 'de', [
...
+	'妙蛙种子',
+	'大尾狸',
+	'ボルケニオン'
+]);
+
+test('Get ID by Japanese name', testIDs, 'jp', [
+	'フシギダネ',
+	'ビーダル',
+	'ボルケニオン'
+]);
```
Comment (sindresorhus) [inline test.js]: We don't need the name test for all languages. We're already testing that code with English and German. We just need the other ID test to ensure the language works, so these two tests can be dropped.

**Comment by twada** [comment]
For Japanese Pokémon names, looks good to me 👍

**Comment by bennyn** [comment]
Thank you so much @cherry-geqi. 

You can also update the [Supported languages](https://github.com/sindresorhus/pokemon#supported-languages) section in `readme.md`, as part of your pull request.

Happy to see many languages coming in! :+1:

**Comment by sindresorhus** [comment]
Thanks @cherry-geqi. I fixed the feedback for you when merging ;)

**Pull request merged**